CLDR-15196 common order for grammatical features

macchiati opened the pull request:
CLDR-15196

- [ ] This PR completes the ticket.

<!--
Thank you for your pull request.
Please see http://cldr.unicode.org/index/process for general
information on contributing to CLDR.

1. Make sure the ticket is filed at
https://unicode-org.atlassian.net/projects/CLDR/
2. Update the PR title and first line of this
message to include the ticket ID (CLDR-_____)
3. You will be automatically asked to sign the contributors’
license before the PR is accepted.
- sign: https://cla-assistant.io/unicode-org/cldr
- license: http://www.unicode.org/copyright.html#License
-->

srl295 commented on common/annotations/dsb.xml:
```diff
@@ -832,7 +832,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<annotation cp="🧑‍⚕" type="tts">gójc/gójcowka</annotation>
 		<annotation cp="👨‍⚕">doktor | gójc | gójcojski kitel</annotation>
-		<annotation cp="👨‍⚕" type="tts"/>
+		<annotation cp="👨‍⚕" type="tts">doktor</annotation>
```
Is this the TTS err issue we have been seeing only sometimes?

macchiati commented:
This is ready for review, Annemarie. I cleaned up the ordering a bit, which you can see from tools/cldr-code/src/main/java/org/unicode/cldr/util/GrammarInfo.java, and also added the reasoning behind the ordering.

(Just noticed a typo in the comments and will fix that.)

srl295 reviewed (approved):
DUgCET - the DUCET for grammar

AEApple commented:
Can we add a note in the Survey Tool about the ordering of the cases to inform the vetters about the ordering?

macchiati commented:
Yes, whenever we push to production we need to make sure that we update the
documentation (and send out a note pointing to the change). We'll want to
try it out on smoketest first, of course!

Mark


On Thu, Dec 2, 2021 at 11:54 AM Annemarie Apple ***@***.***>
wrote:

> Can we add a note in the Survey Tool about the ordering of the cases to
> inform the vetters about the ordering?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/unicode-org/cldr/pull/1641#issuecomment-984953344>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACJLEMDVUIBRYTI75NMKKO3UO7FJDANCNFSM5JFRFE7Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

macchiati commented:
Yes, common/annotations/dsb.xml
<https://github.com/unicode-org/cldr/pull/1641#discussion_r761416911>: was
for the test failure (other failures are hidden in non-exhaustive mode).

Mark


On Thu, Dec 2, 2021 at 12:00 PM Mark Davis ☕️ ***@***.***> wrote:

> Yes, whenever we push to production we need to make sure that we update
> the documentation (and send out a note pointing to the change). We'll want
> to try it out on smoketest first, of course!
>
> Mark
>
>
> On Thu, Dec 2, 2021 at 11:54 AM Annemarie Apple ***@***.***>
> wrote:
>
>> Can we add a note in the Survey Tool about the ordering of the cases to
>> inform the vetters about the ordering?
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/unicode-org/cldr/pull/1641#issuecomment-984953344>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ACJLEMDVUIBRYTI75NMKKO3UO7FJDANCNFSM5JFRFE7Q>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>>
>

macchiati commented:
If someone reviews, then I can start checking it out on Smoketest.

macchiati commented:
Gakk, had to squash because of a bad commit message... so asking for another review. (Really wish squashing didn't require rereview.)

srl295 commented:
> Yes, common/annotations/dsb.xml <[#1641 (comment)](https://github.com/unicode-org/cldr/pull/1641#discussion_r761416911)>: was for the test failure (other failures are hidden in non-exhaustive mode). Mark
> […](#)

*ideally* that would be in the commit message also.

macchiati commented:
I merged this in and checked it out on Smoketest. Noted some problems:
https://unicode-org.atlassian.net/browse/CLDR-15196?focusedCommentId=163433
Mark


On Thu, Dec 2, 2021 at 12:30 PM Steven R. Loomis ***@***.***>
wrote:

> Yes, common/annotations/dsb.xml <#1641 (comment)
> <https://github.com/unicode-org/cldr/pull/1641#discussion_r761416911>>:
> was for the test failure (other failures are hidden in non-exhaustive
> mode). Mark
> … <#m_-7431605127268789373_>
>
> *ideally* that would be in the commit message also.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/unicode-org/cldr/pull/1641#issuecomment-984977658>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACJLEMCNRTPY32CS3U575TDUO7JPVANCNFSM5JFRFE7Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
